openassessment/xblock/openassessmentblock.py

```diff
@@ -336,11 +336,22 @@ def file_upload_type(self, value):
         """
         self.file_upload_type_raw = value
 
+    def get_allowed_file_types_or_preset(self):
```

Comment:
I am trying to wrap my head around terminology. Preset is refers to something we define / hard code and "allowed file type" is something the user defines if they choose "custom" ?

Reply:
Yes. Presets are allowing learners to upload `image` or `pdf-and-image` files, vs `custom` which allows course staff to specify what types of files learners can upload.